iocage/lib/NetworkInterface.py

```diff
@@ -21,32 +21,40 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
+import typing
 import iocage.lib.helpers
 
 
 class NetworkInterface:
+
     ifconfig_command = "/sbin/ifconfig"
     dhclient_command = "/sbin/dhclient"
     rtsold_command = "/usr/sbin/rtsold"
 
-    def __init__(self,
-                 name="vnet0",
-                 ipv4_addresses=[],
-                 ipv6_addresses=[],
-                 mac=None,
-                 mtu=None,
-                 description=None,
-                 rename=None,
-                 addm=None,
-                 vnet=None,
-                 jail=None,
-                 extra_settings=[],
-                 auto_apply=True,
-                 logger=None):
+    def __init__(
+        self,
+        name: typing.Optional[str]="vnet0",
+        create: typing.Optional[bool]=False,
```

Comment:
again, `Optional[bool]` — what's the difference between `False` and `None`?

Reply:
Nothing. If the value is `None`, it becomes `False`.